invesalius/data/brainmesh_handler.py

```diff
@@ -14,6 +14,22 @@ def __init__(self, image, mask, n_peels, affine_vtk, window_width, window_level)
         self.window_width = window_width
         self.window_level = window_level
 
+        flip = vtk.vtkImageFlip()
```

Comment:
this solves the problem, could you please check that normals are not flipped, because the shades of gray on the brain peel is considerably darker than without the flip

Reply:
I think it's better now, I set specular to 0.25 and it's brighter now.